packages/react-core/src/next/components/Wizard/WizardToggle.tsx

```diff
@@ -97,10 +88,10 @@ export const WizardToggle = ({
       >
         <span className={css(styles.wizardToggleList)}>
           <span className={css(styles.wizardToggleListItem)}>
-            {activeStep?.name}
+            {currentStep?.name}
```

Comment:
Missing the toggleNum element here 

https://github.com/patternfly/patternfly-react/blob/af40c12e9dc55ed508ebc5e07415ec93d61bd639/packages/react-core/src/components/Wizard/WizardToggle.tsx#L83

This is what it should look like

<img width="601" alt="Screen Shot 2022-10-04 at 5 02 01 PM" src="https://user-images.githubusercontent.com/35148959/193938516-c49e334a-bd8d-41ec-9ddd-5e5122a0a6bb.png">

This is what it looks like now

<img width="570" alt="Screen Shot 2022-10-04 at 5 01 51 PM" src="https://user-images.githubusercontent.com/35148959/193938539-9019a8f0-1db3-42da-8dad-6795e7c1a2a1.png">

Reply:
Ah I think I lost this when resolving merge conflicts in or from the previous Wizard-next PR, will add this back, good catch!